ibis/expr/api.py

```diff
@@ -421,8 +418,12 @@ def _memtable_from_dataframe(
 ) -> Table:
     from ibis.backends.pandas.client import DataFrameProxy, PandasInMemoryTable
 
+    def _generate_unique_table_name():
+        # create case-insensitive uuid4 unique table name
+        return f"_ibis_memtable_{np.base_repr(uuid.uuid4().int, 36)}"
```

Comment:
```suggestion
        return f"_ibis_memtable_{np.base_repr(uuid.uuid4().int, 36).lower()}"
```

Let's use lowercase for consistency.